docs/src/guide/plugins/cors.md

```diff
@@ -0,0 +1,46 @@
+# CORS Middleware
+
+CORS providing a Jiny middleware that can be used to enable CORS with various options.
+
+Example:
+
+```java
+import com.jinyframework.middlewares.Cors;
+
+public class CorsExample {
+    public static void main(String[] args) throws IOException {
+        val server = HttpServer.port(1234);
+
+        server.use("/all", Cors.newHandler(), ctx -> HttpResponse.of("cors enabled for all"));
+
+        server.use("/console",
+                   Cors.newHandler(Config.builder()
+                                         .allowAll(false)
+                                         .allowOrigin("https://code.google.com")
```

Comment:
Also this is a mistake from me. It should be allowOrigin(<your current domain in the browser where you open the console>).